packages/plugin-commands-publishing/src/pack.ts

```diff
@@ -124,7 +124,9 @@ async function packPkg (destFile: string, filesMap: Record<string, string>, proj
     }
     const mode = isExecutable ? 0o755 : 0o644
     if (/^package\/package\.(json|json5|yaml)/.test(name)) {
-      const publishManifest = await exportableManifest(projectDir, manifest)
+      const readmePath = Object.values(filesMap).find(name => name.toLowerCase().includes('readme.md'))
```

Comment:
use a more strict check
```suggestion
      const readmePath = Object.values(filesMap).find(name => /^package\/readme.md$/i.test(name))
```

Reply:
Do we remove the `toLowerCase` check?

Reply:
it is a regex that ignores case (`/i`)